src/Config/ConfigUpdaterInterface.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\commerce\Config;
+
+/**
+ * Performs configuration updates on behalf of an extension.
+ *
+ * Service to provide methods for extensions to import, revert, or delete
+ * configuration once they have been installed. Intended to be run from
+ * hook_post_update_NAME(), since this involves the modification of entities.
+ *
+ * @see hook_post_update_NAME()
+ */
+interface ConfigUpdaterInterface {
+
+  /**
+   * Imports configuration from extension storage to active storage.
+   *
+   * @param string[] $config_names
+   *   An array of configuration names.
+   *
+   * @return \Drupal\commerce\Config\ConfigUpdateResult
+   *   Object containing failed and succeeded configuration object update.
+   */
+  public function import(array $config_names);
+
+  /**
+   * Reverts configuration to the value from extension storage.
+   *
+   * @param string[] $config_names
+   *   An array of configuration names.
+   * @param bool $skip_modified
+   *   Whether to skip modified configuration, defaults to TRUE. If this is
```

Comment:
The description after "Whether to skip modified configuration" is redundant (obvious from context)